ES/ESmy020.xml

```diff
@@ -31,71 +31,394 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <idno>MY-020</idno>
                </msIdentifier>
                <msContents>
-                  <summary>I) Maṣḥafa gǝnzat “Book of the Funeral Ritual” (fols. 2ra l.1–53rb
-                     l.10)&#xD; - Homily of St Jacob of Serug on funeral of priests and deacons
-                     (fols. 11rb l.3–15rb l.6)&#xD; - Homily of ʾAbbā Salāmā, Metropolitan of
-                     Ethiopia (fols. 32rb l.10–35rb l.10)&#xD; II) Maṣḥafa krǝstǝnnā “Book of the
-                     Christening”, called here Maṣḥafa ṭǝmqat (fols. 54ra l.1–71rb l.8). For the
-                     title of the book s. e.g., EMML 3107 (EMML VIII, 94)&#xD; &#xD; Additiones
-                     1-2.</summary>
+                  <summary>
+                  </summary>
+
+
                   <msItem xml:id="ms_i1">
+                    <locus from="1r" to="74rb"/>
+                    <title type="complete" ref="LIT1931Mashaf"/>
 
-                     <incipit>
-                        <foreign xml:lang="en"/>
-                     </incipit>
-                     <explicit>
-                        <foreign xml:lang="en"/>
-                     </explicit>
-                     <note/>
+                    <msItem xml:id="ms_il.1">
```

Comment:
there is an error in the numbering, you used the letter L (ms_il.1) instead of the figure 1 (ms_i1.1) here and everywhere else, this file is not valid

Reply:
Oh, indeed! Devil pushed my hand :) corrected